services/server/CHANGELOG.md

```diff
@@ -2,6 +2,22 @@
 
 All notable changes to this project will be documented in this file.
 
+## sourcify-server@3.0.0 - 2025-04-09
+
+- Integrate new lib-sourcify classes in server (#1960) 
+- Use VerificationExport for storeVerification methods of storage services (#1995)
+- Improve express error handler (#2027)
+- Add upgrade contract private endpoint (#2042)
+- Add updated_at column and trigger to sourcify_matches (#2047) 
```

Comment:
Shouldn't this be a database change?

Reply:
It's also in database: https://github.com/ethereum/sourcify/pull/2066/files#diff-8eeedd40ba7e8cc0520e02b63f8c639416c1a85f08bf660ac3a585f765d5141dR3-R5

but it's also part of server from my pov

Reply:
okay, let's leave it like this :)